views/membership.erb

```diff
@@ -0,0 +1,261 @@
+            <li><p>Bezplatné služby pre Vaše podnikanie</p></li>
+          </ul>
+
+        <p class="lead"><a href="/clenovia">Zoznam členov</a></p>
+
+        <h2 class="content-heading text-center">Typy členstva:</h2>
+       </div>
+    </div>
+  </div>
+</div>
+
+<div class="section main-content">
+  <div class="panel with-nav-tabs panel-default">
+    <div class="container container-offset">
+      <div class="row">
+      <div class="sidebar col-md-3"></div>
+        <div class="col-md-9">
+          <div class="panel-heading">
+            <ul id="audience-nav-tabs" class="nav nav-tabs">
+              <li class="active">
+                <a href="#tab1default" data-toggle="tab">
+                  <div class="text-center"><span>Základný člen<i class="glyphicon glyphicon-chevron-up"></i></span></div>
+                </a>
+              </li>
+              <li>
+                <a href="#tab2default" data-toggle="tab">
+                  <div class="text-center"><span>Malá firma<i class="glyphicon glyphicon-chevron-up"></i></span></div>
+                </a>
+              </li>
+              <li>
+                <a href="#tab3default" data-toggle="tab">
+                  <div class="text-center"><span>Stredná firma<i class="glyphicon glyphicon-chevron-down"></i></span></div>
+                </a>
+              </li>
+              <li>
+                <a href="#tab4default" data-toggle="tab">
+                  <div class="text-center"><span>Liga výnimočných<i class="glyphicon glyphicon-chevron-down"></i></span></div>
+                </a>
+              </li>
+              <li>
+                <a href="#tab5default" data-toggle="tab">
+                  <div class="text-center"><span>TOP člen<i class="glyphicon glyphicon-chevron-down"></i></span></div>
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="panel-body">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-14">
+            <div class="tab-content">
+            <div class="tab-pane fade" id="tab1default">
+                <div class="row">
+                  <div class="col-md-3"></div>
+                  <div class="col-md-9 tab-pane-content">
+                    <div class="row">
+                      <div class="col-md-6">
+                        <h3>Benefity</h3>
+                        <p>Kategória určená pre fyzické osoby:</p>
+                        <ul>
+                          <li><p>Účasť na chode o.z.</p></li>
+                          <li><p>Meno v zozname členov</p></li>
+                          <li><p>Networking akcie</p></li>
+                          <li><p>Informovanosť o eGovernmente</p></li>
+                        </ul>
+                      </div>
+                      <div class="col-md-3">
+                        <h3>Ekosystém</h3>
+                        Datahub <p>/</p>
```

Comment:
Ak tu tie benefity nie su, tu sekciu tam ani neukazujme. S tymi lomitkami to teraz vyzera trochu akoby sme to tam zabudli dat:

![image](https://user-images.githubusercontent.com/1374745/31383945-34832e8e-adbd-11e7-96be-13d413c0c0a8.png)